### ISSUE TITLE
Problems getting semantic:ui-css on Meteor 1.4.x

### ISSUE THREAD

**Issue by chompomonim** [Semantic-Org/Semantic-UI-CSS#23]
$ meteor add semantic:ui-css
 => Errors while adding packages:             
                                              
While checking for semantic:ui-css@2.2.4:
error: No compatible binary build found for this package. Contact the package author and ask
them to publish it for your platform.

**Comment by cheesington** [comment]
I'm having this problem too.  Which is funny -- I thought that meteor 1.4 was doing away with binaries published by package authors.  At any rate, I have a compatible toolchain installed, as I was able to ```meteor npm install bcrypt``` without trouble.

**Comment by cheesington** [comment]
A workaround was posted on #20, which is to set a previous version for now:

`meteor add semantic:ui-css@=2.1.2`

**Comment by jlukic** [comment]
Ironically I'm also getting this same message with some other packages I depend on in `1.4.x`

**Comment by apendua** [comment]
@jlukic Which packages exactly?

**Comment by apendua** [comment]
For what it's worth, here's the `package.js` file that I am using to define a local smart package and I can fetch whatever version of `semmantic-ui-css` from npm, and it works perfectly well.
```javascript
/*globals Package, Npm*/
Package.describe({
  name        : 'essentials:semantic-ui',
  version     : '2.2.4',
  description : 'A wrapper for semantic-ui',
});

Npm.depends({
  'semantic-ui-css': '2.2.4',
});

Package.onUse(function (api) {
  api.versionsFrom('1.4');
  api.use('jquery', 'client');
  api.addFiles([
    '.npm/package/node_modules/semantic-ui-css/semantic.js',
    '.npm/package/node_modules/semantic-ui-css/semantic.css',
  ], 'client');
  api.addAssets([
    '.npm/package/node_modules/semantic-ui-css/themes/default/assets/fonts/icons.eot',
    '.npm/package/node_modules/semantic-ui-css/themes/default/assets/fonts/icons.otf',
    '.npm/package/node_modules/semantic-ui-css/themes/default/assets/fonts/icons.svg',
    '.npm/package/node_modules/semantic-ui-css/themes/default/assets/fonts/icons.ttf',
    '.npm/package/node_modules/semantic-ui-css/themes/default/assets/fonts/icons.woff',
    '.npm/package/node_modules/semantic-ui-css/themes/default/assets/fonts/icons.woff2',
  ], 'client');
});
```

**Comment by apendua** [comment]
I looked into [package.js](https://github.com/Semantic-Org/Semantic-UI-CSS/blob/master/package.js) and what I think is wrong with it are the following two points:

1. We should be using `addAssets` instead of `addFiles` for fonts and images.
2. Using `versions('1.0')` can force and outdated versions of some packages, which is probably the main source of the problem described in this ticket. Look [here](https://github.com/Semantic-Org/Semantic-UI-CSS/blob/master/.versions).

**Comment by DimitryDushkin** [comment]
Could you please fix this?

**Comment by samuelwight** [comment]
Yeah, the package won't install for me either. Having to use v2.1.2 in order for it to work. :/

**Comment by s-devaney** [comment]
how do i fix this, I need some 2.2 features

**Comment by apendua** [comment]
@s-devaney The best workaround I can think of is creating a local `semantic-ui` package using the code I pasted in my comment above:

https://github.com/Semantic-Org/Semantic-UI-CSS/issues/23#issuecomment-257533357

This will allow you to choose whatever version you want.

**Comment by chompomonim** [comment]
@apendua  why you still need meteor package for that? Pure npm package is not enough?

**Comment by apendua** [comment]
@chompomonim That's a very good question. The only reason I think you will need a package in this particular case is importing font assets. If it was only about `css` and `js` files, you should be able to import them directly into your project with no problems. However, with the current meteor build system, I don't see a clear way (without modifying paths in css) to have those assets included properly without a wrapper package.

**Comment by s-devaney** [comment]
I've tried importing the npm package and I can't get it to work - lots of issues with relative paths to fonts etc not playing nice with the meteor build tool...

**Comment by s-devaney** [comment]
@apendua do you have any instructions on how to add your local smart package to my project?

**Comment by apendua** [comment]
@s-devaney It's great you figured that out. Sorry I couldn't respond earlier, it's a timezone difference problem.

**Comment by gabimoncha** [comment]
check the [last comment](https://github.com/Semantic-Org/Semantic-UI-CSS/issues/38#issuecomment-367984755) on [#38](https://github.com/Semantic-Org/Semantic-UI-CSS/issues/38)
